libs/core/src/components/sage-divider/stories/sage-divider.docs.stories.mdx

```diff
@@ -0,0 +1,58 @@
+import { Meta, Story, Canvas, Source, ArgsTable } from '@storybook/addon-docs';
+import { html, render } from 'lit-html';
+
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+
+import * as stories from './sage-divider.stories.js';
+
+<Meta title="components/Divider" component="sage-divider" argTypes={extractArgTypes('sage-divider')} />
+
+# Divider
+
+Use the divider to separate sections of content. You can also use them to override auto-layout padding.
+
+## Properties
+
+<ArgsTable of="sage-divider" />
+
+### Default
+
+The default divider displays a horizontal line.
+
+<Story story={stories.Default} />
```

Comment:
I might be missing (bad eyesight), but I currently cannot see the dividers at all.
<img width="1030" alt="Screen Shot 2023-04-21 at 4 28 17 PM" src="https://user-images.githubusercontent.com/14791307/233737521-c005fd07-30a9-471c-ad38-bfbe28af9a10.png">

Reply:
@anechol Not at all. Looks like something is def off. I just double-checked and I do indeed see dividers. Sometimes when I switch branches I get odd behaviors. Maybe try restarting?

![Screenshot 2023-04-21 at 2 43 14 PM](https://user-images.githubusercontent.com/1175111/233739244-20769353-50e2-4eea-baaf-04be9301fefa.png)

Reply:
Yep that was def the issue, thanks for reminding me